### ISSUE TITLE
Questionable position of block comments inside if-else block

### ISSUE THREAD

**Issue by maximbaz** [prettier/prettier#5012]
**Prettier 1.14.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAZgAgBTTgSk2AB0pNNIoBnCAGzgDpaIBzbYkPD-AblIF9McWlTiYA9ACpMGHDADuEQpPFFS5SjXpNW7EAEMoEGAAs4AJ258o-EABpOABxipoVZKH3nzEeQAUvBHcUfQA3CFQAE3sQACNzfTAAazgYAGVHRNQoFmQYcwBXOAds0XMYPwSWAFt9ZHR9EWKQACsqAA8AIQTk1LT9argAGWy4esbRB0zzMuQ4-ViAT2YoGMdzbJgAdSjTZAAOAAYpn1EthMc59bgy0LGHczgARwLUR8r9GrqkBqaHUWqqDyhWaVGyLHoAEUCsYxj8Js0YAsdpE9kgAEwOfL6VC0cEAYQg1VqcyMUHuIAKogAKgtgr9RPx+EA)

**Input:**
```jsx
if (one) {
  console.log("one");
} else /* if (two) */ {
  console.log("another");
}
```

**Output:**
```jsx
if (one) {
  console.log("one");
} /* if (two) */ else {
  console.log("another");
}

```

**Expected behavior:**
```jsx
if (one) {
  console.log("one");
} else /* if (two) */ {
  console.log("another");
}
```

-----

I realize that this is a difficult problem to solve and I've seen that you already have quite a number of issues about comments and their positioning, but I didn't see this specific example shown. If it's a duplicate, feel free to close.

Basically I want to be able to temporarily transform `else if` into an `else`. But prettier is doing something unexpected, it rearranges the position of the comment, which makes it difficult to return back to `else if` later on.

**Comment by suchipi** [comment]
Thanks for the report. As a workaround, you can change the conditional to `true || two` instead of using a comment here:
```js
if (one) {
  console.log("one");
} else if (true || two) {
  console.log("another");
}
```

**Comment by lydell** [comment]
I often find it easier to duplicate the line when doing such debugging experiments:

```js
if (one) {
  console.log("one");
  // } else if (two) {
} else {
  console.log("another");
}
```

For me, duplicating lines and commenting out lines are quick editor shortcuts. And I find it easier to edit/delete stuff than adding `/*` and `*/`. As a bonus, I still have the original line around for reference.

Prettier tends to handle lines consisting of only a `//` comment better than inline `/**/` comments.

Not saying that things can't be improved in Prettier's comment handling (but keep in mind that comment handling is super hard), just sharing another approach.
